environment.yml

```diff
@@ -2,7 +2,13 @@ name: cantera-latest
 channels:
   - cantera
   - cantera/label/dev
+  - conda-forge
```

Comment:
I appreciate the intention of this change. However, adding `conda-forge` will (I believe) result in some packages like NumPy and SciPy coming from that channel rather than the `defaults`. In that case, there will be an incompatibility in the compiled clib code between Cantera/Python/NumPy. There is a Cantera package on `conda-forge`, but we don't publish pre-releases there so I don't think that'll work for this purpose. I'm honestly not sure how to continue from here 😦

Reply:
I don't think any of the Jupyter examples require the development version of Cantera at this point, so we could just install everything from `conda-forge`, dropping the `cantera` and `cantera/label/dev` channels entirely.

Reply:
That seems reasonable to me, since the Jupyter examples probably should only contain stuff that is available in the stable release without requiring the user to install the dev version of they want to reproduce the example on their own machine. I'll change `environment.yaml` accordingly.

Reply:
> how serious

In the worst case, I believe they can lead to segmentation faults in the memory which cause an immediate crash of Python. More likely would be that some symbols are named differently or are placed in a different location in the binary, which would also cause errors. Here are some docs from conda-forge about it: https://conda-forge.org/docs/user/tipsandtricks.html#using-multiple-channels